app.js

```diff
@@ -16,9 +17,11 @@ databaseConnect();
 
 app.use(passport.initialize()); 
 
+
 app.use('/books', booksRouter);
 app.use('/user', usersRouter);
 
+corsSetup(app);
```

Comment:
Have you checked that it is working?

Reply:
I checked it with Postman OPTIONS request. The response contained exactly the amount of methods I have defined in my CORS parameters.

Reply:
but have you checked that your existing api works with this options besides OPTIONS request?

Reply:
Yes, I have and it does work as it used to.

Reply:
Sorry, I was wrong, you are the best, I'm noob